packages/language-server/lib/server.js

```diff
@@ -1,18 +1,19 @@
 const {
   createConnection,
-  TextDocuments,
-  ProposedFeatures
+  ProposedFeatures,
+  TextDocumentSyncKind
 } = require("vscode-languageserver");
+const textDocument = require("vscode-languageserver-textdocument");
 
 const { validateDocument } = require("./language-services");
 
 const connection = createConnection(ProposedFeatures.all);
```

Comment:
unrelated to this change, but should we only expose the features we have implemented on the connection?
What is the standard pattern in Language Servers?

Reply:
ok I see this is at pure protocol level, not exposed editor functionality level